packages/components/src/HeaderBar/HeaderBar.test.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import HeaderBarComponent from './HeaderBar.component';
+
+describe('HeaderBar', () => {
+	function t(msgid, options = {}) {
```

Comment:
Nice test additions 👍 
do we not normally use defaultProps for `t` to work in unit tests? I don't really mind though, just wondering